packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -1,20 +1,24 @@
-import cuid from 'cuid'
-import PropTypes from 'prop-types'
-import React from 'react'
-import styled, { css } from 'styled-components'
-import SVGContext from '@plugins/drawingTools/shared/SVGContext'
-import DrawingToolMarks from './components/DrawingToolMarks'
-import TranscribedLines from './components/TranscribedLines'
-import SubTaskPopup from './components/SubTaskPopup'
-
-const StyledRect = styled('rect')`
-  ${props => props.disabled ? 
-    css`cursor: not-allowed;` :
-    css`cursor: crosshair;`
-  }
-`
-
-function InteractionLayer ({
+import cuid from "cuid";
```

Comment:
Style guide: `prettier` is adding semicolons to a lot of these lines; these aren't required.